src/Components/HouseholdDataBlock/HouseholdDataBlock.js

```diff
@@ -37,6 +39,7 @@ const HouseholdDataBlock = ({ handleHouseholdDataSubmit }) => {
     navigate(`/${uuid}/step-${step}/${page}`);
   };
   const ageErrorController = useErrorController(householdMemberAgeHasError, displayHouseholdMemberAgeHelperText);
+  const healthInsuranceErrorController = useErrorController(healthInsuranceDataHasError, getHealthInsuranceError);
```

Comment:
I think that this should be moved to the `HealthInsuranceQ` component

Reply:
I moved the `healthInsuranceErrorController` to the `HealthInsuranceQ` component.